lib/WGwrangler/Model/WireguardDataAdapter.pm

```diff
@@ -0,0 +1,188 @@
+package WGwrangler::Model::WireguardDataAdapter;
+use strict;
+use warnings FATAL => 'all';
+use experimental 'signatures';
+use Mojo::JSON qw(true false);
+
+use base 'Exporter';
+our @EXPORT = qw(get_peer_table_data get_peer_count update_peer_data sort_table_data commit_changes get_section_data disable_peer get_interface_selection);
```

Comment:
I would suggest to only export upon request using `EXPORT_OK` to not randomply pollute the users namespace